packages/twenty-front/src/modules/object-record/record-field/meta-types/display/components/__stories__/perf/DateTimeFieldDisplay.perf.stories.tsx

```diff
@@ -34,7 +34,7 @@ export const Elipsis: Story = {
 
 export const Performance = getProfilingStory({
   componentName: 'DateTimeFieldDisplay',
-  averageThresholdInMs: 0.1,
+  averageThresholdInMs: 0.2,
```

Comment:
TODO: check - if we merge like this we need to be sure there isn't a better solution

Reply:
It is okay to increase if we don't have the choice but in that case you have to "proof" that you've understood the reason why it was increasing and why we can't do any better